Systems/LavaRendering.cs

```diff
@@ -0,0 +1,1154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Terraria;
+using Terraria.GameContent.Liquid;
+using Terraria.Graphics;
+using Terraria.ID;
+using Terraria.ModLoader;
+using static Terraria.WaterfallManager;
+
+namespace CalamityMod.Systems
+{
+    public class LavaRendering
```

Comment:
It seems disingenuous to put a class in the Systems namespace but not have it be a `ModSystem`.

Reply:
tbh it should have been a modsystem from the start. I've changed it to now be a ModSystem to utilize the Load override